src/ansys/dpf/core/meshed_region.py

```diff
@@ -622,12 +628,27 @@ def deep_copy(self, server=None):
             for i, node in enumerate(mesh.nodes.add_nodes(len(node_ids))):
                 node.id = node_ids[i]
                 node.coordinates = coord.get_entity_data(i)
-        with self.elements.connectivities_field.as_local_field() as connect:
-            with self.elements.element_types_field.as_local_field() as types:
-                for i, elem in enumerate(mesh.elements.add_elements(len(element_ids))):
-                    elem.id = element_ids[i]
-                    elem.connectivity = connect.get_entity_data(i)
-                    elem.shape = element_types.shape(types.get_entity_data(i)[0])
+        if len(element_ids) > 0:
+            with self.elements.connectivities_field.as_local_field() as connect:
```

Comment:
@cbellot000 I have a Python question: I'm improving here the `deep_copy` method of the MeshedRegion to also copy the faces PropFields (previously, we ere only copying eltypes and conn, but in a mesh with faces it is also needed to copy the faces structures). Why were we doing these loops in Python (they are quite slow even for small meshes). The approach below for the faces PropFields works:

`m2 = vN[0].meshed_region.deep_copy()`

And then: 

![image](https://github.com/ansys/pydpf-core/assets/107186344/5055753a-18ef-4c63-88d5-b59865b4f477)

Reply:
Changed the behavior for `InProcess` and `gRPC CLayer` to employ the serialize/deserialize operators